0.data-download/scripts/data_loader.py

```diff
@@ -4,28 +4,32 @@
 from random import sample
 from matplotlib import testing
 import pandas as pd
-import numpy 
+import numpy
 from numpy import ndarray
 
-def load_data(data_directory, adult_or_pediatric = "all"):
+
+def load_data(data_directory, adult_or_pediatric="all"):
```

Comment:
should the filenames in this function be changed to account for the 22Q4 release?

Reply:
Yes, I didn't forget! I am waiting for the optimize_hyperparameters.py program to finish running before I change anything since this file is used there and I wasn't sure if it would interrupt the run.